spec/features/checkouts_spec.rb

```diff
@@ -1,9 +1,23 @@
 require 'rails_helper'
 
 RSpec.feature "Checkouts", type: :feature do
+
+  before(:each) do
+  O1 = Order.create(Status: "Completed", total: 23600,
+                   vat: 2100, delivery_cost: 500, user_id: 6, 
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Trailing whitespace detected.